Formula/htslib.rb

```diff
@@ -0,0 +1,26 @@
+class Htslib < Formula
+  desc "C library for high-throughput sequencing data formats"
+  homepage "http://www.htslib.org/"
+  url "https://github.com/samtools/htslib/releases/download/1.6/htslib-1.6.tar.bz2"
+  sha256 "9588be8be0c2390a87b7952d644e7a88bead2991b3468371347965f2e0504ccb"
+  revision 1
+
+  head "https://github.com/samtools/htslib.git"
+
+  depends_on "xz"
```

Comment:
are we sure this is needed?

Reply:
Just tested, it is not. I'll remove it.

Reply:
probably just pass --disable-lzma

Reply:
It isn't strictly required, but it is a fairly small dependency that may increase the utility of this formula.